docs_source/usage/vocabs-unique-keys.md

```diff
@@ -0,0 +1,128 @@
+# A Vocabulary for Identifying Uniqueness of Array Items
+
+## 1. Purpose
+
+This document describes a vocabulary defining keywords that can be used to validate that arrary instances contain unique values as determined by one or more given paths within each item.
+
+The intent for these keywords is to cover the use cases discussed in the JSON Schema vocabularies GitHub issue [#22](https://github.com/json-schema-org/json-schema-vocabularies/issues/22).
+
+## 2. Declarations
+
+The ID for this vocabulary is `https://gregsdennis.github.io/json-everything/vocabs-unique-keys` (the URI to this document).
+
+A draft 2020-12 meta-schema which includes this vocabulary has been defined for convenience.  The ID for the meta-schema is `https://gregsdennis.github.io/json-everything/meta/unique-keys`, and it can also be found at this address.
+
+## 3. The `uniqueKeys` Keyword
+
+### 3.1 Syntax and Semantics
+
+The value of `uniqueKeys` must be an array of JSON Pointers as defined by [RFC 6901](https://tools.ietf.org/html/rfc6901).  At least one JSON Pointer MUST be specified.
+
+These JSON Pointers, when resolved against each item, define a set of values for that item.  If an item does not contain the location specified by a JSON Pointer, the implementation MUST keep a placeholder to indicate a missing value.
+
+Validation for this keyword passes if the value sets for all of the items are distinct, that is, no two sets contain the same collection, including missing values.
+
+Implementations SHOULD support JSON Pointers in both the basic and URI formats as specified by RFC 6901.  URI-formatted JSON Pointers are not required in this context, but they are so frequently used throughout JSON Schema that it makes sense for implementations to support them here as well.
```

Comment:
By "URI formatted JSON Pointer" do you mean something like `#/foo/bar`? That's not a JSON pointer anymore though, but a fragment-only uri reference. `#/foo/bar` won't pass the 'json-pointer' format validation, which is what your metaschema uses.

Reply:
May have a misunderstanding here, but I believe that URI formatted pointers are specified in that RFC (section 6, I think), and they follow the same format.  Basically I want to say that both a raw pointer and the format commonly used in JSON Schema should be accepted.

Reply:
Okay, that's what I said/thought.
I think it's weird to accept both, but at any rate, your schema won't validate against that. You'd have to say `"anyOf": [ { "format": "json-pointer"}, {"format":"uri-reference", pattern: "^#/"} ]` (I threw in the pattern for extra clarity).